storage/build.gradle

```diff
@@ -5,18 +5,21 @@ dependencies {
   implementation project(':ssz')
   implementation project(':util')
 
-  implementation 'org.apache.tuweni:tuweni-kv'
-  implementation 'org.apache.tuweni:tuweni-bytes'
-  implementation 'org.apache.tuweni:tuweni-ssz'
-  implementation 'com.fasterxml.jackson.core:jackson-databind'
   compile 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+  compile group: 'org.mapdb', name: 'mapdb', version: '3.0.7'
```

Comment:
This PR just moves around the ordering, but I'll raise a seperate PR to move the version into `version.gradle` for consistency.